sdk-python/livescoreapi.py

```diff
@@ -10,11 +10,50 @@
 
 
 class livescores:
+    """
+    A class used to get livescores. 
+    ...
+
+    Attributes
+    ----------
+    key : str
+        an alphanumeric string used to access data from the LiveScore API
+    secret : str
+        an alphanumeric string used to access data from the LiveScore API
+    
+    Methods
+    -------
+    _get_html_file(link)
+    
+    get_all()
+    
+    """
+    def __init__(self,key,secret):
+        self.key = key 
+        self.secret = secret 
+    
+    def _get_html_file(self,link):
+        """returns the source code given a link"""
+    
+        try:
+            html_file = urllib.request.urlopen(link)
+            return html_file
+        except Exception as e:
+            print(str(e))
+            
+    def get_all(self):
+        """returns all the livescores"""
+        
+        link = "http://livescore-api.com/api-client/scores/live.json?key={}&secret={}".format(self.key,self.secret)
+        html_file = self._get_html_file(link)
```

Comment:
This itself is not an html file it is a `json_string` or response. To me this whole loop thing should be in the `_get_html_file()` method and the `_get_html_file ` mehtod should return the dictionary with the response ready for manipulating. This way we will reuse the loop and write it only ones. and I would maybe call the method `_make_api_call`, `_do_api_call`, `_call_api`, `_api_call` something like that because get HTML file might mean that you are getting it from the disc and not from the server